### DISCUSSION TITLE
Taiko Editor - Have newly placed notes on top of an already placed note replace it/change color instead of being placed on top.

### DISCUSSION THREAD

**Discussion by XeZey** [ppy/osu#28998] [Ideas] [answered]
Currently, one can put 2 or more notes on the same exact beat, which is pointless. I believe there is some stuff done with sliders/swells in some special maps, but I dont think this has ever or can be done in any intersting way with multiple notes.
This would likely improve the editing of taiko maps quite a bit. One would not need to change from the "select" to "hit" tool if they want to change colors and could instead just change that note by clicking on it with the type selected that they want to change it to, and place new notes as they see fit. they would only have to switch to the select tool if they want to move/delete notes.

**Comment by bdach** [comment, marked as answer]
will be coming to all rulesets
